notebook/static-src/edit/js/main.js

```diff
@@ -95,3 +84,8 @@ require([
     // On document ready, resize codemirror.
     $(document).ready(_handle_resize);
 });
+
+}).catch(function(err) {
+    console.error('Could not load globals', err);
```

Comment:
If we know bootstrap is loaded at this point. Otherwise you could fall back to an alert, stating to open up the dev console? It's in no-recovery-mode at this point anyway.